### DISCUSSION TITLE
Which Stripe Events to send when using Pay with Stripe Checkout? (as described in GoRails April '21 episode)

### DISCUSSION THREAD

**Discussion by k2director** [pay-rails/pay#418] [Q&A] [answered]
Hi! I've got Stripe Checkout via the Pay gem working in Development, and now I'm trying to get it working in Production. I'm trying to create a Webhooks Endpoint via the Stripe dashboard, and it wants to know what Events it should be sending to the end point.  There are a lot of them to choose from! ;-)

Since I'm using Stripe Checkout, should I just choose to send events only for Checkout and the Billing Portal? Those are the two Checkout features I'm using in my app via the Pay gem. I would think so, but there are separate options to send Charges, and Account and Customer events, and I just wasn't sure if the Pay gem would be looking for those...

Thanks very much, Pay has made Checkout very approachable!

**Comment by excid3** [comment, marked as answer]
Here is the current list of webhooks we use from Stripe: https://github.com/pay-rails/pay/blob/master/lib/pay/stripe.rb#L53-L99

You'll need to listen to more than just the Checkout events because we need to record charges and subscriptions in the database.

**Reply by k2director** [reply]
Thanks much, that did the trick!
